### DISCUSSION TITLE
[Bug] Caret not showing

### DISCUSSION THREAD

**Discussion by mdegroux** [microsoft/monaco-editor#4086] [Q&A] [answered]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Link

https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0#XQAAAALwBgAAAAAAAABBqQkHQ5NjdUR-hYwhbNdtBF5u2OZdQGJnnJJ9V4HjI7oBM6FYBPFg2XW9Ig0KvBvhYDgUGEq7tdD_Bvg0OSZ00D9QxKLHgwy2vl-C-knr939cgRdI38tPRhKNpfaXPDxcKRSF7Bk7bxtD_UNlkt_9WETGYIP8R9vOEaYVwEtGbWDy4U2IH4v-aXp73bGLx-RbDhAveBY_sDMlQgtzUA9INLl7-qUUZBT_X_AzK4q5_7L3F7_48z2YUdm1NRAcgqk9ZZvyoT_wVkrohihE-CVMFePG9Xagzr79N4l31a7anNUbWQxTWgStB5kTSiuIEXzpXlEd5AJ1MpnoAkzwpuPAHNivsFxrB0jcho7LpGpHg_5WoHBubD84w9NZzBaT56a3LtsyRy1B7GYdpieNGNl-hnQ9zdVVIdXjQo4InDgwmzYKASU3d0KAOgkn-Pr9i30YJWwiUV21uOF1ejISY3vWSWtmZpD4R6eNt0afFN2xIg_3vy6c2lRy6N_7UIC0SIKYjTVGNR95DOEqKbkI9nuW8by25W1RCCPObK0QmKRuo6w4lcrgUl0Yf_PxY2FshsVRjdnQHebCpNyGKDMsCQhVPkBkX31fSdqN5HLdtNtQ5a5rUciJFuhHXnGgK6zkMK5UKU0LKN5WkJ_zRktS8efJjZ0ayMeiqTjVV0FmNgoggg6bMuglg_IQtUstLWoAPBEKbMnk28TX6f1MfHXq5DLEdqtSH_7gH_Mp9sva4p63Ed9l4Xl_N9ATRAL4RudIA6zFeShyw9St5wiIM8JPoCiQzrIeOLi-Cox0W1G5H7_buuB9ueIduxbK6cHL7GNpS2TABWpQHc38U1-bilMhSEm53D-gR8ZwhyfXhxb3B_r0aCVLf3DAQYQLAsHwBGdn4oMA6NIUh2Gp-tauSpgEjPXIutGWnzyohoyqRR-xyoHeS-o2xpiSv-iAiG9vqew9r72oNRp-FnT20lyrXNIsLCXlSsr8lr_YfgiGcHat42d0eBCSxFPlePZkG3bfkImz2wW3lbJtvDLT8w91GU1DH6zA-CYhgLKcR7n576iG6J0xceVwNG2r3nYQ0HOta2blYqLOXAtIRLo42kerIReDe9hQTiCaIp9NouNR4_V4HoxJb_D7-FW1__-2X_9A

### Reproduction Steps

Expand one of the rows in the grid
And click in the editor, the caret is not showing

### Actual (Problematic) Behavior

When the editor is embedded in in ag-grid, I can't type in.

The grid supports fields, if I replace monaco by a regular input, I can type in

### Expected Behavior

To be able to type

```[tasklist]
### Tasks
```

**Comment by hediet** [comment, marked as answer]
This looks like a problem of `ag-grid-enterprise`.

Can you look into what is going on? Does the editor get focus? Maybe there are css conflicts.

**Reply by mdegroux** [reply]
Thanks a lot, was a focus issue
